### PR TITLE
Update / Add build and runtime configuration files

### PR DESCRIPTION
**Pull request by razvand**
Update QEMU and Firecraker build and runtime configuration files.

### CODE REVIEW

File: .config.helloworld-qemu-aarch64
Comment (RaduNichita) [inline .config.helloworld-qemu-aarch64:1]: Please update the README to fit this change.

File: .config.helloworld-qemu-x86_64
Comment (RaduNichita) [inline .config.helloworld-qemu-x86_64:1]: Please update the README to fit this change.

File: .config.helloworld-fc-x86_64
Comment (RaduNichita) [inline .config.helloworld-fc-x86_64:1]: Please update the README with instructions to build / run the application using firecracker.

**Review by RaduNichita — CHANGES_REQUESTED** [review]
It looks very good, @razvand ! I left some minor nitpicks.

Furthermore, I have a question. When running on firecracker, I see some strange arguments:

```text
Powered by
o.   .o       _ _               __ _
Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
oOo oOO| | | | |   (| | | (_) |  _) :_
 OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
                  Atlas 0.13.1~325daa41
Hello world!
Arguments:  "reboot=k" "panic=1" "pci=off" "nomodules" "8250.nr_uarts=0" "i8042.noaux" "i8042.nomux" "i8042.nopnp" "i8042.dumbkbd"
```

Should we display those? I don't know if they are related to Unikraft.

**Comment by razvand** [comment]
> Should we display those? I don't know if they are related to Unikraft.

I think they are related to Firecracker, so I chose not to display them.

**Review by RaduNichita — APPROVED** [review]
All good from my side now. Thanks @razvand 

Reviewed-by: Radu Nichita <radunichita99@gmail.com>

**Review by StefanJum — APPROVED** [review]
All good, thanks.

Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request closed without merging**